crypto/err/openssl.ec

```diff
@@ -40,7 +40,7 @@ L SM2           NONE                            crypto/sm2/sm2_err.c
 L OSSL_STORE    include/openssl/storeerr.h      crypto/store/store_err.c                include/crypto/storeerr.h
 L ESS           include/openssl/esserr.h        crypto/ess/ess_err.c                    include/crypto/esserr.h
 L PROP          NONE                            crypto/property/property_err.c          include/internal/propertyerr.h
-L PROV          NONE                            providers/common/provider_err.c         providers/common/include/prov/providercommonerr.h
+L PROV          include/openssl/proverr.h       providers/common/provider_err.c         providers/common/include/prov/providercommonerr.h
```

Comment:
While you're cleaning up, I would shorten the name of the internal header.  `proverr.h` should suffice, considering that the inclusion would be `#include "prov/proverr.h"`
(as a matter of fact, it could even be shortened to `err.h`, making the inclusion `#include "prov/err.h"`, but I can see how some might not be able to stomach being so non-verbose :wink:)